test/writeFunctions.test.ts

```
@@ -394,6 +394,18 @@ describe("Write Functions Client Tests", () => {
         hatId: hatX_1_2,
         newEligibility: agreementInstance2,
       });
+
+      // set hats as claimable in MCH for signAgreementAndClaim tests
+      // ClaimType.ClaimableFor = 2
+      const mchModule = hatsModulesClient.getModuleById(MCH_MODULE_ID) as Module;
+      await walletClient.writeContract({
+        address: mchInstance,
+        abi: mchModule.abi,
+        functionName: "setHatsClaimability",
+        args: [[hatX_1_1, hatX_1_2], [2, 2]], // ClaimType.ClaimableFor = 2
```

ESLint

[error] Replace `[hatX_1_1,·hatX_1_2],·[2,·2]` with `⏎··········[hatX_1_1,·hatX_1_2],⏎··········[2,·2],⏎········`

(prettier/prettier)